### ISSUE TITLE
Telework: Form submission not sending emails

### ISSUE THREAD

**Issue by MeghanStothers** [bcgov/digital-journeys#1180]
As a form submitter, I must be able to send my submission to many manager for approval

**Comment by MeghanStothers** [comment]
![image.png](https://images.zenhubusercontent.com/61ddb02eac135d772cfce6ec/7d74c3ac-2ccb-4520-855f-f6516ec448f5)

**Comment by MeghanStothers** [comment]
link to digital form removed and alert message add to the page: https://www2.gov.bc.ca/gov/content/careers-myhr/all-employees/work-arrangements/flexible-workplaces/get-started-with-flexible-work-in-the-bc-public-service/submitting

digital-journeys.apps.silver.devops.gov.bc.ca/form

**Comment by iman-jamali-fw** [comment]
@MeghanStothers @Stella-Archer 
Unfortunately, we have not yet figured out what the issue with the Telework workflow issue is We're not able to replicate the issue on our local environments -- making it harder to debug the issue. 

At this point I'd suggest keeping the link to Telework disabled during the weekend, to prevent broken submissions for users. Hopefully, we can fix it on Monday.

**Comment by MeghanStothers** [comment]
I've let the form owner and Alison know and we'll tackle this one Monday. Thanks for your quick action.

**Comment by bhumin-fw** [comment]
To identify, we have followed the below steps.
1. Restart services in a specific order. That refresh all tokens used to communicate between internal services.
2. Make sure the latest workflow and form are up. We have updated the same.
3. Here, We found that the form works fine, but the workflow does not create on Camunda.
4. Added log for the same to see which requests are failing. In the Camunda and Webapi.
5. Run the same separately via Postman and find the below point.
6. In the latest release, we have added a required field in the supervisor section. Those are hide in the form that the user submits.
7. Move the validation in form as a dynamic script. So applies validation for the visible form/components only.

Outcome:
The dynamic script requires to set. It will apply validation for hidden fields when those are visible.

For example,
In telework. 
The user submits the form and the supervisor reviews it.
Users and supervisors can fill out different fields. Fields that are filled by the supervisor are hidden from the user.  These fields are only visible if the supervisor opens this form in the "review and process form" tab. 
Now validation of such fields that are accessible by the supervisor needs to handle dynamically using the script. It will apply validation when supervisor open a form in review tab.
